editor/export/editor_export_platform_pc.cpp

```diff
@@ -237,9 +239,8 @@ void EditorExportPlatformPC::set_logo(const Ref<Texture2D> &p_logo) {
 }
 
 void EditorExportPlatformPC::get_platform_features(List<String> *r_features) const {
-	r_features->push_back("pc"); //all pcs support "pc"
-	r_features->push_back("s3tc"); //all pcs support "s3tc" compression
```

Comment:
Removed this, which is redundant with the above logic, and also wrong if users for some reason decide to export ETC2/ASTC and not S3TC/BPTC.